src/components/Nav/Nav.js

```diff
@@ -1,8 +1,52 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import './Nav.scss';
 
+import nikeLogo from '../../assets/Nav/nike.png';
+
 const Nav = () => {
-  return <div className="nav">nav</div>;
+  const [navData, setNavData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/navData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setNavData(data);
+      });
+  });
+
+  return (
+    <div className="nav">
+      <div className="navContainer">
+        <Link className="nikeLogo" to="/">
+          <img alt="nikeLogo" src={nikeLogo} />
+        </Link>
+        <div className="navText">
+          {navData.map(info => {
```

Comment:
```suggestion
          {navData.map(({id,url,menu})=> {
```
구조분해할당 바로 해주면, 불필요한 객체 입력이 줄어듭니다.

Reply:
피드백 주신 부분 수정 완료 후 현재 PR에 commit으로 추가했습니다 ☺️